Giovaniavs/MyProject-server
# Feature(feature/dockerfile): configuring Dockerfile

## What I did
* Created the Dockerfile

## How to test
1. Run the following command in your terminal: **docker build -f Dockerfile -t myproject/container .**
2. Then, **docker run -d -p 1234:3001 myproject/container**
3. Finally, try to access localhost:1234 in your browser and see if there any message right there
